exponent_server_sdk/__init__.py

```diff
@@ -227,12 +243,14 @@ def validate_response(self):
         if self.details:
             error = self.details.get('error', None)
 
-            if error == PushResponse.ERROR_DEVICE_NOT_REGISTERED:
+            if error == PushReceiptResponse.ERROR_DEVICE_NOT_REGISTERED:
```

Comment:
makes sense as a change, could consider making these constants only available in one class but I think this is probably better